src/instrumentation_measurement/measurements.py

```diff
@@ -50,9 +51,10 @@ def measure(self, sampleSize, evm="geth", nSamples=1):
 
     geth = "geth"
     openethereum = "openethereum"
+    openethereum_wasm = "openethereum-wasm"
```

Comment:
can we do consistently `openethereum_ewasm` (underscore, "e") or is this spelling used anywhere else?